src/components/HeroSection/HeroSection.module.css

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
If you decided in the previous PRs to put these imports in the ```index.css``` file (which is the better way imo), then lets keep on doing that and add this import there.